sw/device/silicon_creator/rom_ext/doc/rescue.md

```diff
@@ -124,6 +127,18 @@ ok: reboot
 
 The ROM_EXT will then exit rescue mode and reboot the chip.
 
+#### Disable automatic reboot (`WAIT`)
+
+The user may request that the ROM_EXT disable the automatic reboot after upload actions with the 4-byte code `WAIT`.
+The ROM_EXT will acknowledge this request with the following message:
+
+```
+mode: WAIT
+ok: wait after upload
+```
+
+Once commanded to `WAIT`, the ROM_EXT will need an explicit reboot after an upload action.
```

Comment:
Does this apply only to the very next upload action or all uploads until next reset?  (It would be a strange use case for sending firmware twice in a single rescue session.)

Reply:
It disables automatic reboot after upload actions until the next reset.  A mutli-upload scenario is possible during an ownership transfer operation:

- Upload next_owner config.
- Upload next_owner firmware.
- Upload boot_svc OwnershipActivate command
- Reboot